auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div id="business-lookup" v-if="state !== States.SUMMARY">
+    <v-autocomplete
+      append-icon="mdi-magnify"
+      autocomplete="chrome-off"
+      autofocus
+      class="mt-5 mb-n2"
+      filled
+      hint="For example: &quot;Joe's Plumbing Inc.&quot;, &quot;BC1234567&quot;, &quot;FM1234567&quot;"
+      label="Business Name or Incorporation/Registration Number"
+      no-filter
+      persistent-hint
+      return-object
+      v-model="selectedBusiness"
+      :hide-no-data="state != States.NO_RESULTS"
+      :items="searchResults"
+      :loading="state == States.SEARCHING"
+      :name="Math.random()"
+      :search-input.sync="searchField"
+    >
+      <!-- NB: use empty selection slot to stop re-triggering selectedBusiness watcher -->
+      <template v-slot:selection="" />
+
+      <template v-slot:prepend-item>
+        <p class="pt-2 pl-5 font-size-12">
+          Active B.C. Businesses
+        </p>
+      </template>
+
+      <template v-slot:no-data>
+        <p class="pl-5 font-weight-bold">
+          No active B.C. business found
+        </p>
+        <p class="pl-5">
+          Ensure you have entered the correction business name or numbers.
+        </p>
+      </template>
+
+      <template v-slot:item="{ item }">
+        <v-row class="business-lookup-result pt-1">
+          <v-col cols="2" class="result-identifier">{{item.identifier}}</v-col>
+          <v-col cols="8" class="result-name">{{item.name}}</v-col>
+          <v-col cols="2" class="result-action">
+            <span v-if="item.disabled" class="added">Added</span>
+            <span v-else class="select">Select</span>
+          </v-col>
+        </v-row>
+      </template>
+    </v-autocomplete>
+  </div>
+</template>
+
+<script lang="ts">
+import { BusinessLookupIF, BusinessLookupResultIF, EmptyBusinessLookup } from '@/models'
+import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
+import BusinessLookupServices from '@/services/business-lookup.services'
+import { Debounce } from 'vue-debounce-decorator'
+import Vue from 'vue'
+import { namespace } from 'vuex-class'
+
+const BusinessModule = namespace('business')
+
+enum States {
+  INITIAL = 'initial',
+  SEARCHING = 'searching',
+  SHOW_RESULTS = 'show results',
+  NO_RESULTS = 'no results',
+  SUMMARY = 'summary'
+}
+
+@Component({})
+export default class BusinessLookup extends Vue {
```

Comment:
This is almost the same as the shared component, but with some UI differences. It would be a medium-size job to combine them, so I didn't.

Reply:
Hey, class isn't bad :)

Yes, a single code source would be better, but I didn't want to get into that here (and I don't want to refactor Create UI or Edit UI either).

How about if I create a tech debt ticket?

Reply:
It's ok, we have a bunch of those already hahaha

Reply:
yeah that's fine with me 👍

Reply:
New tech debt ticket is https://github.com/bcgov/entity/issues/16821.